cluster/terraform_kubernetes/config/development.tfvars.json

```diff
@@ -14,5 +14,6 @@
   "welcome_app_hostnames": [
     "www.cluster1.development.teacherservices.cloud"
   ],
-  "ingress_nginx_version": "4.8.3"
+  "ingress_nginx_version": "4.8.3",
+  "enable_prometheus": true
```

Comment:
Can we change to enable_monitoring so it can be used for the other monitoring components?

Reply:
done